x/auth/client/query.go

```diff
@@ -41,22 +41,12 @@ func QueryTxsByEvents(clientCtx client.Context, events []string, page, limit int
 		return nil, err
 	}
 
-	prove := !clientCtx.TrustNode
-
-	resTxs, err := node.TxSearch(query, prove, page, limit, orderBy)
+	// TODO: this may not always need to be proven
```

Comment:
What are these TODO's for? Either reference a GH link or remove them 👍

Reply:
add GH issue link under readme.